src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -671,4 +676,21 @@ public TeamDetailsBundle getTeamDetailsForStudent(StudentAttributes student) {
         return null;
     }
 
+    /**
+     * lines is the enrollment lines entered by the instructor.
+     */
+    private String addDuplicateEmailInfo(String sanitizedLine, String[] linesArray, int isDuplicate) {
```

Comment:
why is there the word `add` for this method, when the other helper methods do not have it?